### PR TITLE
Update ddd.java

### PR DESCRIPTION
**Pull request by codingswag998**
changed printed value

### CODE REVIEW

File: src/ddd.java [hunk 1]
```diff
@@ -3,6 +3,6 @@
 
 	public static void main(String args[]){
 		
-		System.out.print("Hello world");
```
Comment (ElmTim) [inline src/ddd.java:6]: Why do we need this change?

Comment (codingswag998) [reply src/ddd.java:6]: This is needed because we want to change a value

Comment (ElmTim) [reply src/ddd.java:6]: OK

**Comment by ElmTim** [comment]
Ok

**Pull request merged**